package.json

```diff
@@ -56,5 +56,6 @@
 		"postcss-nesting": "latest",
 		"rollup": "^2.10",
 		"rollup-plugin-terser": "^7"
-	}
+	},
+	"sideEffects": ["./src/spaces/index.js"]
```

Comment:
Should we also add `src/index.js` here?

Reply:
Yes, it does seem to have side effects 👍